code/app/src/androidTest/java/com/example/goldencarrot/OrganizerEventTest.java

```diff
@@ -42,8 +43,7 @@ public void testEventCreation() {
             onView(withId(R.id.createEventButton)).perform(click());
 
             // Check for success message or behavior
-            onView(withId(R.id.successMessageTextView))  // Ensure this ID exists
-                    .check(matches(withText("Event created successfully")));
+            onView(withText("Welcome back ")).check(matches(isDisplayed()));
```

Comment:
why did we change this to welcome back? Event created successfully is more informative